src/ArborX_Config.hpp.in

```diff
@@ -12,7 +12,14 @@
 #ifndef ARBORX_CONFIG_HPP
 #define ARBORX_CONFIG_HPP
 
+// ARBORX_VERSION % 100 is the patch level
+// ARBORX_VERSION / 100 % 100 is the minor version
+// ARBORX_VERSION / 10000 is the major version
 #cmakedefine ARBORX_VERSION @ARBORX_VERSION@
+#cmakedefine ARBORX_VERSION_MAJOR @ARBORX_VERSION_MAJOR@
+#cmakedefine ARBORX_VERSION_MINOR @ARBORX_VERSION_MINOR@
+#cmakedefine ARBORX_VERSION_PATCH @ARBORX_VERSION_PATCH@
```

Comment:
Heads up, we discovered in Kokkos that this was a mistake and that a 0 version number would yield an undefined macro